packages/migrate/src/migrate.ts

```diff
@@ -101,13 +86,13 @@ export async function* migrate(input: MigrateInput): AsyncGenerator<string> {
     sys.readFile(filePath, encoding)
   );
 
-  const { options, fileNames: someFiles } = parseJsonConfigFileContent(
-    config,
-    ts.sys,
-    dirname(configFile),
-    {},
-    configFile
-  );
+  const parsed = parseJsonConfigFileContent(config, ts.sys, dirname(configFile), {}, configFile);
```

Comment:
@cafreeman  can we move this outside of migrating the package to keep it independent from any framework-specific details?

Reply:
This actually existed prior to the glint stuff, but I did move all the glint-specific functions to a `glint-utils` file like we discussed